src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let total;
+  const firstNum = Big(numberOne);
+  const secondNum = Big(numberTwo);
+
+  if (operation === '+') {
+    total = firstNum.plus(secondNum);
+  } else if (operation === '-') {
+    total = firstNum.minus(secondNum);
+  } else if (operation === '/') {
+    total = firstNum.div(secondNum);
+  } else if (operation === '*') {
+    total = firstNum.times(secondNum);
+  } else if (operation === '%') {
+    if (secondNum) {
+      total = (firstNum.times(0.01)) * secondNum;
```

Comment:
### Heads-Up:
You don't have to do the calculating yourself, that where the use of `Big` comes in -> [big js library](http://mikemcl.github.io/big.js/#mod)

example: case modifier will be -> `firstNum.mod.(secondNum)`